### ISSUE TITLE
F20: Delete `epo:refersToLot` relation originating from the Notice instance

### ISSUE THREAD

**Issue by csnyulas** [OP-TED/ted-rdf-mapping#415]
In ContractModificationNotice, generated from form F20, remove the `epo:refersToLot` relationship linking a Lot instance to the Notice instance. This is unnecessary (because one can get to the lot through the Notice `epo:announcesContract` Contract `epo:includesLot` Lot property path), and in case of the ContractModificationNotice it is also conceptually wrong (as `epo:refersToLot` should be used only on ResultNotices).

![image](https://github.com/OP-TED/ted-rdf-mapping/assets/2441046/d5a35f90-98e0-41a2-9b4b-1d4c1ba410f2)

**Comment by csnyulas** [comment]
Closing this, as the solution, and the generated output have been checked, and they are correct.

**Comment by muricna** [comment]
@csnyulas @costezki  According to version 2.1.1.rc1 the conceptual and technical mapping still goes through Notice refersToLot which is not in line with the ontology.

**Comment by csnyulas** [comment]
Indeed, there was a mistake in `2.1.1.rc1`, as we removed the **wrong row** ("epo:Procedure epo:hasProcurementScopeDividedIntoLot epo:Lot" instead of "epo:Notice epo:refersToLot epo:Lot")  from the **Conceptual Mapping of F20** . However, the technical mapping was correct. I.e. there is no relationship `epo:refersToLot` between the  `epo:Notice` and `epo:Lot` instances in any of the F20 outputs. As specified in the description of this GH issue, the problem was discovered in F20, which used to have the `epo:refersToLot` relationship between `epo-not:ContractModificationNotice` and `epo:Lot` instances, although, according to EPO 3.1.0, the  `epo:refersToLot` relation links only the `epo-not:ResultNotice` class to the `epo:Lot` class.

So, this issue was not about removing the `epo:refersToLot` relation between **all** `epo:Notice` and `epo:Lot` instances, therefore, removing the relation in other forms, such as F03, F06, etc. was not necessary. _As long as they are of type `epo-not:ResultNotice`._ It is true that our solution is incomplete, as in case of some forms that can represent also notices that are not of type `epo-not:ResultNotice`, e.g. `epo-not:PlanningNotice` or `epo-not:CompetitionNotice` (such as forms F21 or F22), we still create this relationship. However, those were not covered by the scope of the current mappings, and will be fixed later.
I created a separate issue for that: https://github.com/OP-TED/ted-rdf-mapping/issues/430

**Comment by csnyulas** [comment]
Closing this as CM was updated (see linked commit), and will be available in `2.1.1.rc2`.
The other issue #430 will e fixed separately.

**Comment by muricna** [comment]
The lot could be reached by going throught the procedure

**Comment by csnyulas** [comment]
Of course. One can write a more complex query, involving the `epo:refersToProcedure` and `epo:hasProcurementScopeDividedIntoLot` properties to get to the lots mentioned in a notice.
Or a simpler one, to query the lots directly from the ResultNotice (using the `epo-not:refersToLot` property). 
**EPO 3.1.0 allows it.** Isn't it? So, what is wrong? **Why is this ticket reopened?** 

![image](https://github.com/OP-TED/ted-rdf-mapping/assets/2441046/22077fdd-7e9c-4dc5-9c26-cd0835348956)

Screenshot from: https://docs.ted.europa.eu/EPO/3.1.0/_attachments/html_reports/eNotice/index.html

BTW, I just checked, and this property is also in EPO 4.0.0-rc.1. So, it should be totally legit, and I'd say appropriate and useful, to use it:
https://github.com/OP-TED/ePO/blob/release/4.0.0-rc.1/implementation/eNotice/owl_ontology/eNotice_owl_core.ttl#L574

Again. Can you please explain why it is wrong what it is done, and why did you reopen the ticket?
